lib/rules/jsx-boolean-value.js

```diff
@@ -30,15 +34,25 @@ function getErrorData(exceptions) {
   }
   return errorData.get(exceptions);
 }
-
+/**
+     * @param {string} configuration
+     * @param {Set<any>} exceptions
+     * @param {string} propName
+     * @returns {boolean} propName
+  */
 function isAlways(configuration, exceptions, propName) {
   const isException = exceptions.has(propName);
   if (configuration === ALWAYS) {
     return !isException;
   }
   return isException;
 }
-
+/**
+     * @param {string} configuration
```

Comment:
all these blocks need to be deindented a bit

Reply:
Done. @ljharb